app/services/Tracing.ts

```diff
@@ -17,20 +9,17 @@ import {
 
 export const HoneycombConfig = Config.nested("HONEYCOMB")(
   Config.all({
-    apiKey: Config.secret("API_KEY"),
+    apiKey: Config.string("API_KEY"),
```

Comment:
Is there any equivalent of `Config.secret` now?

Reply:
Yep, you should use `Config.redacted`

https://effect.website/docs/configuration/#secret